spring/spring-boot-4/it-tests/embedded-tests/pom.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+   <modelVersion>4.0.0</modelVersion>
+
+   <parent>
+      <groupId>org.infinispan</groupId>
+      <artifactId>infinispan-spring-boot4-starter-parent</artifactId>
+      <version>16.0.0-SNAPSHOT</version>
+      <relativePath>../../pom.xml</relativePath>
+   </parent>
+
+   <name>Infinispan Spring Boot 4 Starter Embedded IT</name>
+   <artifactId>infinispan-spring-boot4-starter-embedded-tests</artifactId>
+   <dependencies>
+      <dependency>
+         <groupId>org.infinispan</groupId>
+         <artifactId>infinispan-spring-boot4-starter-embedded</artifactId>
+      </dependency>
+      <dependency>
+         <groupId>org.springframework.boot</groupId>
+         <artifactId>spring-boot-starter-web</artifactId>
+      </dependency>
+      <dependency>
+         <groupId>org.springframework.boot</groupId>
+         <artifactId>spring-boot-starter-actuator</artifactId>
```

Comment:
Depending what you do in those ITs you may want to get rid of the actuator if it's not needed. In Spring Boot 4 you can expose metrics without the actuator.